README.md

```diff
@@ -43,9 +43,22 @@ out, running through [some tutorials](https://google.github.io/xls/tutorials/),
 [reporting bugs](https://github.com/google/xls/issues), and letting us know
 what you think!
 
+## Install using conda
+
+```bash
+curl -O https://repo.anaconda.com/miniconda/Miniconda3-py37_23.1.0-1-Linux-x86_64.sh
+bash Miniconda3-py37_23.1.0-1-Linux-x86_64.sh -p conda-env/ -b
```

Comment:
OOC does this only work for Python 3.7?

Reply:
since include `python 3.7`, I wanted to default on this since some of the other conda-eda packages only support 3.7 right now, see https://github.com/hdl/conda-eda/issues/72 and https://github.com/hdl/conda-eda/issues/260

Reply:
Can you include at the start of the section in the markdown whether these instruction will /only/ work for Python 3.7? e.g. right now I have Python 3.10 and I'm not sure if this means "needs at least 3.7" or " needs exactly 3.7".

If it's "exactly 3.7" but we anticipate being able to support more point-versions if we could point at a GitHub issue that'd be swell.

Reply:
Sorry I misspelled my previous comment, what I meant is that the miniconda installer by default *includes* its own distribution of python 3.7, there is no requirements on the version of python installed on the host (could be none or different, it wouldn't affect the packages executed from the conda environment, and wouldn't be applicable here anyway since we're only packaging the tool executables)

Reply:
So the reason I'm using 3.7 here is to ensure better compatiblity with the *others* conda-eda packages, but we could default on https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh if we didn't want to be opiniated here.

Reply:
reverter to Miniconda3-latest-Linux-x86_64.sh i

Reply:
Thanks for explaining!